synapse/lib/ast.py

```diff
@@ -1195,6 +1195,10 @@ async def yieldFromValu(self, runt, valu):
                     yield node
             return
 
+        if isinstance(valu, s_stormtypes.Prim) and hasattr(valu, 'node'):
```

Comment:
Def not into this type of rigged inspection. Lets discuss, maybe we can make a special subclass / method or something :)

Reply:
updated to add a nodes method on Prim which can be overridden